### DISCUSSION TITLE
Link to another file

### DISCUSSION THREAD

**Discussion by azuravian** [maforget/ComicRackCE#59] [Ideas]
So, to keep some of my organizational sanity, I have been creating fileless books for issues that are in another collection.

For example, Thunderbolts (2006) has multiple collected volumes, but issues 130-131 have never been collected in a Thunderbolts titled collection.  Instead, they are in a Deadpool collection.

My current workaround is creating a fileless book, series: Thunderbolts, issues: 130-131 and having the Title be "In Deadpool By Daniel Way: Vol. 1" so at least I know where to go in order to find the issues as I'm reading.

What I would love, if possible, is the ability to link to another file from a fileless book.  This way I could either double-click (if supported) or right click and follow the link to open the book that actually contains those issues.  Extra super-duper bonus points if you could specify a page number in the linked book and it would open to that page.

**Comment by maforget** [comment]
There is already a Link to File in Fileless comics. Well then it isn't a fileless anymore.

![image](https://github.com/maforget/ComicRackCE/assets/11904426/209a2116-74c5-4742-8202-bef32f1238a2)

Do you mean redirecting to another entry in the library? There is nothing stopping you linking it to another file that you already have added, but it won't be another entry.

**Comment by azuravian** [comment]
Yeah, I had tried that.  When you link to file, you get an error message "This file is already used in the Library".  Basically, what I want is the ability to treat a fileless comic as its own object in the database that has a link to another object in the database.
